lib/Newsletter/ViewInBrowser.php

```diff
@@ -2,12 +2,19 @@
 namespace MailPoet\Newsletter;
 
 use MailPoet\Models\Setting;
-use MailPoet\Models\Subscriber;
 use MailPoet\Newsletter\Links\Links;
 use MailPoet\Newsletter\Renderer\Renderer;
 use MailPoet\Newsletter\Shortcodes\Shortcodes;
 
 class ViewInBrowser {
+
+  /** @var bool */
+  private $is_tracking_enabled;
+
+  function __construct($is_tracking_enabled) {
```

Comment:
How is this working? I don't see passing the argument anywhere.

Reply:
It is passed here: https://github.com/mailpoet/mailpoet/pull/1766/commits/e7dfdf1cdbfd354ed1084a53f4d1f13f6b6b711e#diff-a565a70a1dc7bccd9aec5961b86160f9R35